test/e2e/run-all.js

```diff
@@ -13,6 +13,15 @@ const getTestPathsForTestDir = async (testDir) => {
   return testPaths;
 };
 
+// TODO: Simplify?
```

Comment:
This seems reasonably simple enough to me already! I tried a few alternatives and found nothing simpler.
```suggestion
```

Reply:
Oh, but there is this: https://lodash.com/docs/4.17.15#chunk

That would be simple.

Reply:
If you think it's simple enough to read as is I would prefer not importing a dep for it 😄